aiourlshortener/shorteners/base.py

```diff
@@ -35,10 +39,8 @@ def expand(self, url: str) -> str:
 
     @coroutine
     def close(self):
```

Comment:
we can use `@atexit.register` decorator to avoid func param confusion in init method

Reply:
```python
    @atexit.register
    @coroutine
    def close(self):
        if self._session is not None:
            yield from self._session.close()
```
and
```python
    @coroutine
    @atexit.register
    def close(self):
        if self._session is not None:
            yield from self._session.close()
```
cause:
```
Error in atexit._run_exitfuncs:
TypeError: close() missing 1 required positional argument: 'self'
```

Reply:
I think coroutine functions can not be called from atexit.register

Reply:
I think `@coroutine` functions can not be called from `atexit.register`